.github/workflows/release.yaml

```
@@ -0,0 +1,121 @@
+name: "Release"
+run-name: Release ${{ inputs.tag }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      revision:
+        type: string
+        description: "commit sha1 revision where to start the release, must be the long commit sha1"
+        required: true
+      tag:
+        type: string
+        description: "the new tag/release version to create"
+        required: true
+
+env:
+  RELEASE_BRANCH: feature/release-${{inputs.tag}}
```

Let's keep current release branch name

```suggestion
  RELEASE_BRANCH: release-${{inputs.tag}}
```

---

Done!